src/webhooks/overview.mdx

```diff
@@ -0,0 +1,74 @@
+---
+name: Overview
+menu: WEBHOOKS
+route: /docs/webhooks/overview
+---
+
+import { Title, Code, TextBlock } from "components"
+import webhook_settings from 'assets/webhook_settings.png'
+
+<Title>Overview</Title>
+Webhooks are sent from anomalies. Anomalies are described <a href="https://updates.smartlook.com/worried-about-anomalies-in-your-events-190172" target="_blank">HERE</a>
```

Comment:
Tady bych to napsal spis s nejakym lehcim intrem do toho, co vlastne webhooks jsou:

Webhooks are essential part of our API. In opposite to the REST API, that returns data upon a request, webhooks deliver data to you proactively whenever something happens in your project. Currently the webhooks can be triggered by [anomalies](https://updates.smartlook.com/worried-about-anomalies-in-your-events-190172) in your data.